### ISSUE TITLE
Error: IllegalArgumentExceptionChildren: duplicate children added

### ISSUE THREAD

**Issue by TioCoding** [edvin/tornadofx#548]
Hello, I'm doing an application with multiple views, in my main view I have several buttons that navigate between all the views. The problem occurs when I call a view that inside has a TabPane. and I get the following message:

```
java.lang.IllegalArgumentException: Children: duplicate children added: parent = JFXButton@6bb9eb67[styleClass=button jfx-button]''
	at javafx.scene.Parent$2.onProposedChange(Parent.java:454) ~[jfxrt.jar:na]
	at com.sun.javafx.collections.VetoableListDecorator.add(VetoableListDecorator.java:317) ~[jfxrt.jar:na]
.
..
...
at com.clario.ClarioDesktopApp.view.cliente.ClienteScreen$1$1$2$1$1.invoke(ClienteScreen.kt:60) ~[classes/:na] // Here points to my TabPane
at com.clario.ClarioDesktopApp.view.cliente.ClienteScreen$1$1$2$1$1.invoke(ClienteScreen.kt:19) ~[classes/:na]
```

According to the message the error is when I add my a tab:
```
tabpane {
  tab(ListDataProductoScreen::class) // According to the message, the error is here, but it is an empty view
}
```

Now, reviewing the code, the JFXButton that duplicates this in my main view, in the header of my main view. But I do not understand why the error is there or because it is duplicated, if I do not change or change anything in that part.
```
this += JFXButton().apply {
  graphic = MaterialDesignIconView(MaterialDesignIcon.CHEVRON_DOUBLE_LEFT)
}
```

This is the implementation of my button that supposedly duplicates, but as I say, there is nothing that affects my button, it is simply in my header of my main view. And the error mysteriously appears when I open a subview that inside has a TabPane. Thank you

**Comment by TioCoding** [comment]
I was in a problem to find the error, and I was surprised that the error was when I added the icon to my button.
`                                        graphic = MaterialDesignIconView(MaterialDesignIcon.CHEVRON_DOUBLE_LEFT)`
Why does the error come out? and how can I solve it?

**Comment by TioCoding** [comment]
For some strange reason, the problem was in the CSS, my buttons were added to a class and the property was "-fx-content-display", with the value of "graphic-only". 
```
.button-header{
    -fx-content-display: graphic-only; // It produced the error and it was eliminated
}
```
At first I did not think that the css produced the error, but it is already solved, delete that property from the class and it worked perfectly. Sorry for creating the Issue, i thought TornadoFx had to do with this. Regards.
